pkg/asset/rhcos/image.go

```diff
@@ -82,6 +82,12 @@ func osImage(config *types.InstallConfig) (string, error) {
 	case azure.Name:
 		osimage, err = rhcos.VHD(ctx)
 	case baremetal.Name:
+		// Check for RHCOS image URL override
+		if osimage = config.Platform.BareMetal.ClusterMachineOSImage; osimage != "" {
+			logrus.Infof("Found an URL override in Baremetal Platform for Cluster Machine OS Image: %s", osimage)
+			return osimage, nil
```

Comment:
https://github.com/openshift/installer/pull/2757/files#r355028568

Reply:
OK. Will change it.